frontend/src/app/modules/home/modals/tiles/data/tiles-modal-data.ts

```diff
@@ -122,5 +148,16 @@ export class TilesModalData {
             this.granularityDropdownOneHour, this.granularityDropdownOneDay];
         this.dateRangeGranularityMap[TileDateRangeType.ThePast2Weeks] = [
             this.granularityDropdownOneHour, this.granularityDropdownOneDay];
+        this.dateRangeGranularityMap[TileDateRangeType.ThePast2Weeks] = [
+            this.granularityDropdownOneHour, this.granularityDropdownOneDay];
+        this.dateRangeGranularityMap[TileDateRangeType.ThePast2Weeks] = [
+            this.granularityDropdownOneHour, this.granularityDropdownOneDay];
+
+        this.dateRangeHeatMapGranularityMap[TileDateRangeType.ThePastWeek] = [
+            this.granularityDropdownOneDay];
+        this.dateRangeHeatMapGranularityMap[TileDateRangeType.ThePast30Days] = [
+            this.granularityDropdownOneWeek];
+        this.dateRangeHeatMapGranularityMap[TileDateRangeType.ThePastYear] = [
+            this.granularityDropdownOneMonth];
```

Comment:
Why do you need 1 granularity per DateRange?

Reply:
In future versions, it will be possible to easily add new functionality

Reply:
Now you can just hide it